low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -23,13 +23,22 @@ import sys
 import shutil
 import json
 import subprocess
+import logging
 
 from framework.base.sspl_constants import (PRODUCT_FAMILY, SUPPORT_REQUESTOR_NAME,
     SUPPORT_EMAIL_ID, SUPPORT_CONTACT_NUMBER, ENCL_TRIGGER_LOG_MAX_RETRY,
     ENCL_DOWNLOAD_LOG_MAX_RETRY, ENCL_DOWNLOAD_LOG_WAIT_BEFORE_RETRY, CONSUL_DUMP,
-    DATA_PATH)
+    DATA_PATH, SB_LOG_FILE_PATH)
 from framework.utils.conf_utils import Conf, SSPL_CONF
 
+# set Logging Handlers
+logger = logging.getLogger('sspl_support_bundle')
+logger.setLevel(logging.DEBUG)
+f_handler = logging.FileHandler(f"{SB_LOG_FILE_PATH}/sspl_support_bundle.log")
+f_format = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
```

Comment:
this formatting should be same sspl logging.

Reply:
Done.